gnuradio-runtime/swig/CMakeLists.txt

```diff
@@ -74,7 +74,7 @@ GR_PYTHON_INSTALL(FILES ${CMAKE_CURRENT_BINARY_DIR}/runtime_swig.py
 install(EXPORT runtime_swig-export
   FILE runtime_swigTargets.cmake
   NAMESPACE gnuradio::
-  DESTINATION ${GR_CMAKE_DIR}
+  DESTINATION ${GR_LIBRARY_DIR}/cmake/gnuradio
```

Comment:
why are you reverting this? it should -not- be GR_LIBRARY_DIR, since that's not where the other exports are installed!

Reply:
I accidentally changed this back when I copied the working file I had for the PR when I opened the issue.  I will change this to the other commit and retest on Monday.